src/index.js

```diff
@@ -16,10 +16,9 @@ export default function App () {
           </Route>
         </Routes>
       </BrowserRouter>
-   <Services />
-    <Footer />
```

Comment:
Hey we just talked about this for the other PR, here we can see more clearly the footer is being unexpectedly removed. Please make sure the footer is included 👍